Dockerfile

```diff
@@ -15,7 +15,9 @@ WORKDIR /usr/picard
 
 # Build the distribution jar, clean up everything else
 RUN ./gradlew shadowJar && \
+    ./gradlew cloudJar && \
     mv build/libs/picard.jar picard.jar && \
+    mv build/libs/picardcloud.jar picardcloud.jar && \
```

Comment:
I do agree though that it's weird to package both jars into one Docker, when you'd only ever want one or the other. I'd think it'd be better to have two images: one for the shadowJar, and one for the cloudJar.